client/src/main/scala/drt/client/components/TerminalComponent.scala

```diff
@@ -262,7 +267,7 @@ object TerminalComponent {
 
   val component: Component[Props, Unit, Backend, CtorType.Props] = ScalaComponent.builder[Props]("Loader")
     .renderBackend[Backend]
-    .componentDidMount(p => Callback(SPACircuit.dispatch(GetUserPreferenceIntervalMinutes())) >>
+    .componentDidMount(p => Callback(SPACircuit.dispatch(GetUserPreferences)) >>
```

Comment:
We already fetch the user preferences when the app loads so we don't need it here. We avoid putting requests in `componentDidMount` as components can mount multiple times due to parent & prop updates